tests/test_navmesh.py

```diff
@@ -105,3 +109,33 @@ def test_recompute_navmesh(test_scene, sim):
             some_diff = True
 
     assert some_diff
+
+
+@pytest.mark.parametrize("test_scene", test_scenes)
+def test_navmesh_area(test_scene, sim):
+    if not osp.exists(test_scene):
+        pytest.skip(f"{test_scene} not found")
+
+    cfg_settings = examples.settings.default_sim_settings.copy()
+    cfg_settings["scene"] = test_scene
+    hab_cfg = examples.settings.make_cfg(cfg_settings)
+    sim.reconfigure(hab_cfg)
+
+    # get the initial navmesh area. This test assumes default navmesh assets.
+    loadedNavMeshArea = sim.pathfinder.navigable_area
+    if test_scene.endswith("skokloster-castle.glb"):
+        assert loadedNavMeshArea == 226.65673828125
```

Comment:
These asserts are going to be rather brittle, perhaps we should check if the are equal within an epsilon.